tests/integration.rs

```diff
@@ -4035,4 +4042,135 @@ mod tests {
             Ok(())
         });
     }
+
+    // One thing to note about this test. The virtio-net device is heavily used
+    // through each ssh command. There's no need to perform a dedicated test to
+    // verify the migration went well for virtio-net.
+    #[cfg_attr(feature = "mmio", test)]
+    fn test_migration() {
```

Comment:
My only nit is that it isn't really migration but `test_snapshot_restore`

Reply:
Done :)